packages/tinacms/src/admin/index.tsx

```diff
@@ -237,6 +237,10 @@ export const TinaAdmin = ({
                         }
                       />
                     )}
+                    <Route
```

Comment:
@jamespohalloran What is this used for?

Reply:
Oops! This must have slipped in with something else I was playing with. Removing now